### PR TITLE
OCPBUGS-7427: Associate wait time with method to decrease run time

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

File: frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts [hunk 1]
```diff
@@ -13,13 +13,15 @@ const PLUGIN_PULL_SPEC = Cypress.env('PLUGIN_PULL_SPEC');
 /* The update wait is the value to wait for the poll of /api/check-updates to return with the updated list of plugins
  after the plugin is enabled and loaded. This wait will be longer on ci than when debugging locally. */
 const CHECK_UPDATE_WAIT = 300000;
+const WAIT_OPTIONS = { requestTimeout: CHECK_UPDATE_WAIT };
```
Comment (spadgett) [inline frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts:16]: nit: I'd inline this since it's only used once

**Review by spadgett — APPROVED** [review]
/lgtm

File: frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts (hunk 1 above)
Comment (rhamilto) [reply frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts:16]: Thanks, @spadgett.  I had a version that was reusing this and forgot to clean up.

File: frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts [hunk 2]
```diff
@@ -59,16 +61,12 @@ const enableDemoPlugin = (enable: boolean) => {
...
-    cy.wait('@checkUpdates');
+    cy.wait('@checkUpdates', WAIT_OPTIONS);
     cy.log('Running plugin test locally using bridge.');
   } else {
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(CHECK_UPDATE_WAIT);
     cy.log(`Running plugin test on ci using PLUGIN_PULL_SPEC: ${PLUGIN_PULL_SPEC}`);
   }
-  cy.get(warningToast)
+  cy.byTestID(refreshWebConsoleLink, { timeout: CHECK_UPDATE_WAIT })
```
Comment (jcaianirh) [inline frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts:69]: @rhamilto we've had some issues in the past with the toast notification firing. Do you think we should be testing for the rendering of the toast notification here?

Comment (rhamilto) [reply frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts:69]: Not sure I follow.  This change is to simply use a better selector for the toast notification and to extend the timeout to 5 minutes so the fixed wait is not required.

**Review by jcaianirh — APPROVED** [review]

**Pull request merged**